dspace/config/modules/storage.cfg

```diff
@@ -0,0 +1,31 @@
+#---------------------------------------------------------------#
+#-----------------STORAGE CONFIGURATIONS------------------------#
+#---------------------------------------------------------------#
+# Configuration properties used by the bitstore.xml config file #
+#                                                               #
+#---------------------------------------------------------------#
+
+# Use the localStore or the s3Store implementation
+assetstore.storename.0 = localStore
```

Comment:
I'm not a fan of ending this with `.0` because it implies you can simply add an `assetstore.storename.1` and expect it to work.  It won't work though as if you want multiple they must be specified in `bitstore.xml`.

I'd recommend we change this to better describe the setting to be something like:
```
# Use 'localStore' or 's3Store' as the primary assetstore. 
# These named assetstores are defined in config/spring/api/bitstore.xml
# If you want to add additional assetstores, they must be added to that bitstore.xml
assetstore.storename.primary = localStore
```